src/Activities/Internal/ActivityLoggerHelpers.cs

```diff
@@ -22,7 +22,7 @@ internal static class ActivityLoggerHelpers
 			builder.Append(name).Append(':');
 
 		/// <summary>
-		/// Appends collection of key value pairs to StringBuilder, by calling ToSting on key and value
+		/// Appends collection of key value pairs to StringBuilder, by calling ToString on key and value
 		/// </summary>
 		/// <remarks>
 		/// TODO: should be replaced by AppendJoin when we remove netstandard target, since KeyValuePair also overrides ToString
```

Comment:
@K-Cully what change can be made here now?

Reply:
AppendJoin did not exist in NetStandard.
As far as I understand, this entire method is now redundant and can be dropped in favor of just using AppendJoin https://learn.microsoft.com/en-us/dotnet/api/system.text.stringbuilder.appendjoin?view=net-8.0